torch/_inductor/codegen/triton_utils.py

```diff
@@ -32,17 +30,12 @@ def is_aligned(x):
         if isinstance(x, TensorArg):
             return x.buffer not in V.graph.unaligned_buffers
         if isinstance(x, SizeArg):
-            if isinstance(x.expr, (int, sympy.Integer)):
```

Comment:
Can it be that we're missing some important reason of having this check in place? cc @voznesenskym.

Reply:
I think the old code is just wrong